pkg/security/security_profile/activity_tree/activity_tree.go

```diff
@@ -359,18 +364,23 @@ func (at *ActivityTree) CreateProcessNode(entry *model.ProcessCacheEntry, branch
 	}
 
 	// look for a ProcessActivityNode by process cookie
-	if entry.Cookie > 0 {
+	cs := cookieSelector{}
+	if entry.ExecTime.UnixNano() != 0 && entry.Cookie != 0 {
```

Comment:
I think we should rather use `!entry.ExecTime.IsZero()` if possible 🤔

Reply:
Suggestion applied, thanks